hw/ip/adc_ctrl/data/BUILD

```diff
@@ -6,19 +6,11 @@ package(default_visibility = ["//visibility:public"])
 
 load(
     "//rules:autogen.bzl",
-    "autogen_hjson_c_header",
-    "autogen_hjson_rust_header",
+    "autogen_hjson_sw_headers",
```

Comment:
I thought separating the c and rust headers was useful since different code depends on them, tock for rust and all others for c.

Reply:
They are separated, this is just a macro that creates several targets: one for C, one for rust and one for DT.

Reply:
Got it, thanks.